numpy/_core/meson.build

```diff
@@ -262,7 +264,7 @@ foreach optional_attr: optional_variable_attributes
       return 0;
     }
   '''
-  if cc.compiles(code)
+  if cc.compiles(code, name: optional_attr[0])
```

Comment:
Full discloser, I don't know why this is needed, but it seems fine :)

Reply:
nice solution.